articles/architecture-scenarios/implementation/b2b/b2b-authorization.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Authorization
+description: Understand user authorization and related planning considerations for your B2B implementation.
+toc: true
+topics:
+    - b2b
+    - b2biam
+    - authorization
+contentType: concept
+useCase:
+  - user-authorization
+---
+# Authorization
+
+<%= include('../../_includes/_authorization/_introduction.md', { platform: 'b2b' }) %>
+
+## Application integration
+
+<%= include('../../_includes/_authorization/_application-integration.md', { platform: 'b2b' }) %>
```

Comment:
Missing the extra blurb about including the organization in the ID token: https://docs.google.com/document/d/1pNFH_PkWUA_oiekrAQnN3CFNQr4bzVfUSXkZeEYZO9I/edit#bookmark=id.2vsttna7f084

Reply:
oops - I'll add it